packages/flutter/test/cupertino/context_menu_test.dart

```diff
@@ -264,15 +264,15 @@ void main() {
       Rect decoyChildRect = tester.getRect(findDecoyChild(child));
       expect(childRect, equals(decoyChildRect));
 
-      expect(find.byType(ShaderMask), findsOneWidget);
+      expect(find.byKey(const Key('context-decoy-container')), findsOneWidget);
 
       // After a small delay, the _DecoyChild has begun to animate.
-      await tester.pump(const Duration(milliseconds: 100));
+      await tester.pump(const Duration(milliseconds: 400));
       decoyChildRect = tester.getRect(findDecoyChild(child));
       expect(childRect, isNot(equals(decoyChildRect)));
 
       // Eventually the decoy fully scales by _kOpenSize.
-      await tester.pump(const Duration(milliseconds: 500));
+      await tester.pump(const Duration(milliseconds: 800));
```

Comment:
Is there anything new that's worth testing now?  I guess all of the changes are only visual...  Maybe you could check for the border radius when it's open?

Reply:
Yes, the changes should all be visual. I can add a check for the border radius though 👍